### PR TITLE
INF-224 enable logging for ELK

### PR DESCRIPTION
**Pull request by joaquincasares**
Forward logs to ELK using images currently running in dev environments.

Tested this PR with the new image built with https://github.com/AudiusProject/audius-protocol/pull/3846.

I tested using different machines to run the content-node, discovery-provider, and identity-service and ensure logs were forwarded to ELK.

I also ensured that setting the $audius_loggly_disable flag disabled the container from forwarding logs.

### CODE REVIEW

**Comment by cheran-senthil** [comment]
Is this tested?

**Comment by joaquincasares** [comment]
Not yet.

Which staging node should I test it on and is testing off a audius-docker-compose branch supported? If so, do we have any docs on this?

Or would you recommend I spin up a new machine and test there?

**Comment by joaquincasares** [comment]
@cheran-senthil This has now been tested, the description has been updated, and this PR is ready for review.

File: discovery-provider/docker-compose.yml
Comment (cheran-senthil) [inline discovery-provider/docker-compose.yml:83]: Why are we renaming this, containers will be separated soon

File: discovery-provider/docker-compose.yml
Comment (cheran-senthil) [inline discovery-provider/docker-compose.yml:49]: Possibly a follow up, but we should not be using override.env to manage these env vars, we should use override only for server specific variables consumed by the protocol repo

File: creator-node/docker-compose.yml
Comment (cheran-senthil) [inline creator-node/docker-compose.yml:52]: Same comments as below, we should not be doing multiple tasks in the same PR

File: discovery-provider/docker-compose.yml
Comment (joaquincasares) [reply discovery-provider/docker-compose.yml:83]: Filebeat adds processors and forwards to different indexes by container name as see in `logging/filebeat/filebeat.docker.yml` in the sister PR mentioned in the description:

* https://github.com/AudiusProject/audius-protocol/pull/3846/files

Once containers are separated, we will need a follow up PR to release a Filebeat image.

File: creator-node/docker-compose.yml
Comment (joaquincasares) [reply creator-node/docker-compose.yml:52]: Without these changes the purpose of this PR, to enable logging for ELK, will not be functional.

File: discovery-provider/docker-compose.yml
Comment (joaquincasares) [reply discovery-provider/docker-compose.yml:49]: For sure, feel free to create a follow up ticket and I can handle the reorg you have in mind.

I'm currently following the previously approved method of adding logging containers as seen in #14 .

**Review by cheran-senthil — APPROVED** [review]

**Comment by cheran-senthil** [comment]
Additional audius-cli testing might be necessary, since those uses the name at places

**Comment by cheran-senthil** [comment]
Also, let's not merge this until after this release, since auto upgrade pulls this 🤔

**Comment by SidSethi** [comment]
closing bc stale

**Pull request closed without merging**
